DemoServer/Controllers/Demos/TextSearch/FTSQuerySearchWildcards/metadata.json

```diff
@@ -0,0 +1,193 @@
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'LastFms' collection using the Session `Query()` LINQ method syntax."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type" : "demo",
+                    "Url"  : "/queries/query-overview"
+                },
+                {
+                    "Title": "LINQ Query & Method Syntax",
+                    "Type" : "link",
+                    "Url"  : "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/query-syntax-and-method-syntax-in-linq"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria I",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `Search()` to search for the specified terms in the specified document field.",
+                "",
+                "* In this example,<br>",
+                "  we search for documents that have a term with _'start' param prefix_ AND _'end' param postfix_,",
+                "  within their _Artist_ field.",
+                "",
+                "* Note - as opposed to searching with _wildcard as a postfix_ - <br>",
+                "  searching with _wildcard as the prefix_ of the term (i.e. '*text') is Not recommended,<br>",
+                "  as it will cause RavenDB to perform a full scan instead of an efficient prefix search."
+            ],
+            "Assets": [
+                {
+                    "Title": "FTS Query - Basics Demo",
+                    "Type" : "demo",
+                    "Url"  : "/text-search/fts-query-search-basics"
+                },
+                {
+                    "Title": "FTS Query - Search Operations Demo",
+                    "Type" : "demo",
+                    "Url"  : "/text-search/fts-query-search-operators"
+                },
+                {
+                    "Title": "How to search",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/client-api/session/querying/how-to-use-search"
+                },
+                {
+                    "Title": "Querying searching",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/querying/searching"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria II",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* This step adds an additional `Search()` condition, which is OR'ed with the previous one.",
+                "",
+                "* We search for documents that have a term containing the 'middle' param text in their _Title_ field,<br>",
```

Comment:
Taking into account recommendation from the previous step about not using wildcard as a prefix, should we comment this as well - is it okay to use this variant?

Reply:
you **can** use this `*text*` variant - it will work
but as stated before - using the 'wildcard as a prefix' is less recommended
so I guess I will just add another comment here at this step to remind that